README.md

```diff
@@ -11,27 +11,109 @@ $ npm install --save @coorpacademy/squirrel
 
 
 ## Usage
-### Command Line Interface
 
+### Node interface
+
+```JavaScript
+const squirrel = createSquirrel({
+    hosts: 'http://localhost:2379',
+    auth: null,
+    ca: null,
+    key: null,
+    cert: null,
+
+    cwd: '/',
+    fallback: '/tmp/squirrel.json',
+    indexes: ['foo', 'bar.baz']
+});
 ```
-squirrel-sync --hosts localhost:2379,localhost:2379 ./my-folder /etcd-folder
-squirrel-watch --hosts localhost:2379,localhost:2379 /etcd-folder
+
+Options:
+- `hosts`: ETCD hosts. [more](https://github.com/stianeikeland/node-etcd/#etcdhosts--1270012379-options)
+- `auth`: A hash containing `{user: "username", pass: "password"}` for basic auth. [more](https://github.com/stianeikeland/node-etcd/#constructor-options)
+- `ca`: Ca certificate. [more](https://github.com/stianeikeland/node-etcd/#constructor-options)
+- `key`: Client key. [more](https://github.com/stianeikeland/node-etcd/#constructor-options)
+- `cert`: Client certificate. [more](https://github.com/stianeikeland/node-etcd/#constructor-options)
+- `cwd`: ETCD current working directory.
+- `fallback`: Temporary file to save ETCD replicat.
+- `indexes`: Array of key to index.
+
+#### Methods
+
+Consider
+
+
+```Bash
+/
+├── bar
+│   └── baz   { "bar": { "baz": "qux" } }
+└── foo       { "foo": "bar" }
 ```
 
-### Retrieve a Client
+- `get`
 
+Get file by path. Return `Promise`;
 
+```JavaScript
+const foo = await squirrel.get('/foo');
+console.log(foo); // { "foo": "bar" }
 
-## Test
-You may run test with
+const barBaz = await squirrel.get('/bar/baz');
+console.log(barBaz); // { "bar": { "baz": "qux" } }
 ```
-$ npm test
+
+- `getBy`
+
+Get by index value. Return `Promise`;
+
+```JavaScript
+const foo = await squirrel.getBy('foo', 'bar');
+console.log(foo); // { "foo": "bar" }
+
+const barBaz = await squirrel.getBy('bar.baz', 'qux');
+console.log(barBaz); // { "bar": { "baz": "qux" } }
 ```
 
-Please note that test use an actual etcd service
+Use internally [`_.get`](https://lodash.com/docs#get).
```

Comment:
`Use internally ['_.get']` --> `Field can be nested, as described by ['_.get']` (ou quelque chose du style)

Sinon 'Uses [`_.get`] internally)